app/workers/publish_taxon_worker.rb

```diff
@@ -0,0 +1,10 @@
+class PublishTaxonWorker
+  include Sidekiq::Worker
+
+  def perform(taxon_content_id)
+    Services.publishing_api.publish(taxon_content_id)
+  rescue GdsApi::HTTPClientError => ex
```

Comment:
Edit: Surprised to see there isn't, so I raised [a PR](https://github.com/alphagov/publishing-api/pull/980) against the `publishing-api`. Feel free to add to that thread if you have strong feelings about the HTTP semantics of this response.